File: a215(love_counting)/Python/love_counting.py

while True:
    try:
        start, goal = map(int, input().split())
        sum = 0
        count = 0
        while sum <= goal:
            sum += start
            count += 1
            start += 1
        if count == 0: count = 1
        print(count)
    except:
        break

File: a215(love_counting)/C++/love_counting.cpp
#include <iostream>

int main(void) {
  int start, goal;
  while (std::cin >> start >> goal) {
    int sum = 0, count = 0;
    do {
      sum += start;
      start++;
      count++;
    } while (sum <= goal);
    std::cout << count << '\n';
  }

  return 0;
}
